voxelgpt.py

```diff
@@ -595,9 +595,9 @@ def _label_classes_message(label_classes):
 Hi! I'm VoxelGPT, your AI assistant for computer vision.
 
 I can help you with the following tasks:
-- **Querying your data** 🔎 &nbsp; I can help you filter, match, sort, and more - without writing a line of code. Tell me what you'd like to see and I'll load the corresponding view
-- **Becoming a FiftyOne pro** 💪 &nbsp; I have access to the FiftyOne documentation, so I can help you learn how to use FiftyOne and find the information you're looking for
-- **Troubleshooting data quality** 📈 &nbsp; I can help you build better datasets and higher quality models by answering general knowledge questions about computer vision and machine learning
+- 🔎 **Querying your data** I can help you filter, match, sort, and more - without writing a line of code. Tell me what you'd like to see and I'll load the corresponding view
```

Comment:
Can you add colons after the bold sections please?

Reply:
(the bold doesn't show very well in the App right now)

Reply:
screenshot in PR body